extract_assets.py

```diff
@@ -63,6 +110,8 @@ def main():
     # revision ID in the local asset file.
     new_version = 7
 
+    fileLUT = get_files()
```

Comment:
so if you print something in `get_files` you can see that its running indiscriminately every time that you compile. i already had a `~/roms` directory so its actually slowing the build down quite a bit as well. can you put this after the following try/catch?